pages/[user]/book.tsx

```diff
@@ -2,7 +2,7 @@ import Head from "next/head";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import { CalendarIcon, ClockIcon, ExclamationIcon, LocationMarkerIcon } from "@heroicons/react/solid";
-import prisma from "../../lib/prisma";
+import prisma, {whereAndSelect} from "../../lib/prisma";
```

Comment:
Codacy has a fix for the issue: [Replace `whereAndSelect` with `·whereAndSelect·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
import prisma, { whereAndSelect } from "../../lib/prisma";
```